ui/src/dataLoaders/components/TelegrafEditor.tsx

```diff
@@ -0,0 +1,101 @@
+// Libraries
+import React, {PureComponent} from 'react'
+import {connect} from 'react-redux'
+
+import {ErrorHandling} from 'src/shared/decorators/errors'
+import {AppState} from 'src/types'
+import {Grid, Columns} from '@influxdata/clockface'
+import TelegrafEditorSidebar from 'src/dataLoaders/components/TelegrafEditorSidebar'
+import TelegrafEditorMonaco from 'src/dataLoaders/components/TelegrafEditorMonaco'
+
+import './TelegrafEditor.scss'
+
+interface StateProps {
+  [map: string]: any
+}
+
+interface OwnProps {
+  onJump: any
+  onAdd: any
```

Comment:
I imagine that `onJump` and `onAdd` are functions, why not type them as such?

Reply:
because i forgot to!